modules/storage-persistent-storage-efs-pvc.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-efs.adoc
+
+[id="efs-pvc_{context}"]
+= Creating the persistent volume claim
+
+.Prerequisites
+
+* The EFS provisioner pod has been created successfully.
+
+.Procedure
+
+. In the {product-title} console, click *Storage* -> *Persistent Volume Claims*.
+
+. In the persistent volume claims overview, click *Create Persistent Volume
+Claim*.
+
+. Define the desired options on the page that appears.
+
+.. Select the storage class created previously from the drop-down menu.
+
+.. Enter a unique name for the storage claim.
+
+.. Select the access mode. This determines the read and write access for the
```

Comment:
s/mode. This determines/mode to determine

Is there anything else that a user needs to know before they make this choice?

Reply:
Adjusted the first sentence. The access modes are described under the initial PV storage section, and should be fairly self-descriptive (Read-Write-Once/Read-Write-Many/Read-Only-Many).